applications/flash_attention_v2/collective/xe_flash_attn_softmax_epilogue.hpp

```diff
@@ -87,12 +138,11 @@ template <typename Element> struct Softmax {
       }
     }
   }
-  template <bool CausalMask, int Vec, int FragsM, int FragsN, class FragAcc, class FragMax, class FragSum,
-            class FragOut>
-  CUTLASS_DEVICE static void run(bool is_first, FragAcc &frag_s, FragMax &max, FragSum &sum, FragOut &out,
-                                 Params const &params) {
+
+  template <int Vec, int FragsM, int FragsN, class FragAcc, class FragMax, class FragSum, class FragOut>
+  CUTLASS_DEVICE void operator()(bool is_first, FragAcc &frag_s, FragMax &max, FragSum &sum, FragOut &out) {
```

Comment:
`int Vec, int FragsM, int FragsN,` This can be part of the Epilogue class definition as well since it is getting used in all epilogue function. In this case your operator call does not need to add them as template as well

Reply:
Fixed this.

Reply:
What I meant was that you have enough information in the epilogue to do this calculation, these are compile time calculation. I did not mean to bring them to the use level code. I think that should not be calculated by the user

Reply:
Fixed this.